x-pack/plugins/stack_connectors/server/connector_types/sentinelone/sentinelone.ts

```diff
@@ -134,77 +135,100 @@ export class SentinelOneConnector extends SubActionConnector<
     });
   }
 
-  public async fetchAgentFiles({
-    files,
-    agentUUID,
-    zipPassCode,
-  }: SentinelOneFetchAgentFilesParams) {
-    const agent = await this.getAgents({ uuid: agentUUID });
+  public async fetchAgentFiles(
```

Comment:
@ersin-erdal ,
I read the PR description and just want to make sure that the changes done to these public methods have no impact on consumers of the COnnector's sub-actions - correct? The second argument (`ConnectorMetricsCollector`) is something that the Action task runner will take care of internally?

Also,
Is there a way you can trigger the Security Solution tests (unit) to run on PR?  They only currently run if files from the Security Solution plugin are changed, thus they did not run on this one. If needed, you can make a change (a JS comment would do, I think) to one of our files - perhaps one these:

- https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/endpoint/services/actions/clients/sentinelone/sentinel_one_actions_client.ts
- https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/endpoint/services/actions/clients/crowdstrike/crowdstrike_actions_client.ts


That should trigger CI to run our full test suite.

cc/ @tomsonpl  - just FYI

Reply:
Correct, I intentionally didn't touch any params of the connector types, just passed the `ConnectorMetricsCollector` as a second param.

Connector types just passes it to the request function, so it can collect the metrics there.
Then the Actions plugin will save the data collected by it, in event log.

There shouldn't be any change or impact on the connectors, they should work as they were.

I will add a comment in Security Solution as you suggest. Thanks.